core-bundle/src/Migration/Version500/GuestsMigration.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration\Version500;
+
+use Contao\CoreBundle\Migration\AbstractMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
+use Contao\StringUtil;
+use Doctrine\DBAL\Connection;
+
+/**
+ * @internal
+ */
+class GuestsMigration extends AbstractMigration
```

Comment:
@contao/developers this is another one that looks like it should be an abstract that I can use in my extension. Maybe I did not name it `groups` but I think many extensions have the same issue, don't you think?

Reply:
Abstract migration added in e6647793df121d1a855c9921c3cbad086613ed53.